.github/dependabot.yml

```diff
@@ -2,6 +2,7 @@ version: 2
 updates:
   - package-ecosystem: "gomod"
     directory: "/"
+    open-pull-requests-limit: 0
```

Comment:
Is this supposed to be in the `gomod` section and not in `npm`?

Reply:
aah yup, I didn't put it in the correct section. Sorry about that.
Thanks I will fix it.